spec/Overview-WebCryptoAPI.xml

```diff
@@ -8753,8 +8753,16 @@ dictionary <dfn id="dfn-EcKeyImportParams">EcKeyImportParams</dfn> : <a href="#d
                               <p>
                                 Let <var>key</var> be a new <a href="#dfn-CryptoKey">CryptoKey</a>
                                 object that represents the Elliptic Curve public key identified by
-                                performing the conversion steps defined in Section 2.2 of <a
-                                href="#RFC5480">RFC 5480</a>.
+                                performing the conversion steps defined in Section 2.3.4 of <a
+                                href="#SEC1">SEC 1</a>.
+                                If a decode error occurs or an identity point is found,
+                                <a href="#concept-throw">throw</a> a
+                                <a href="#dfn-DataError"><code>DataError</code></a>.
+                                The uncompressed point format MUST be supported.
+                                If the implementation does not support the compressed point format and
+                                a compressed point is decoded, 
```

Comment:
Suggest s/decoded/provided/, since if compressed is not supported then a compressed point cannot be decoded.